sklearn/distances.py

```diff
@@ -28,4 +36,18 @@
                             goal_outer_loops=params.goal,
                             time_limit=params.time_limit,
                             verbose=params.verbose)
-    print_row(columns, params, function=metric.capitalize(), time=time)
+    if params.output_format == 'csv':
```

Comment:
Since distances benchmark had flow that differs from others, `print_output` wasn't suitable there, but I changed benchmark to work with one metric and now this functions is added.